src/libbson/src/bson/bson.c

```diff
@@ -379,7 +385,7 @@ _bson_append_va (bson_t *bson,              /* IN */
 static bool
 _bson_append (bson_t *bson,              /* IN */
               uint32_t n_pairs,          /* IN */
-              uint32_t n_bytes,          /* IN */
+              uint64_t n_bytes,          /* IN */
```

Comment:
```suggestion
              uint32_t n_bytes,          /* IN */
```

Revert to `uint32_t` as this change is no-longer-needed.

Reply:
Oops, I missed that one. Thanks for catching it.